rkt/cli_apps.go

```diff
@@ -353,3 +353,67 @@ func (ag *appGroup) String() string {
 func (ag *appGroup) Type() string {
 	return "appGroup"
 }
+
+// appCapsRetain is for --caps-retain flags in the form of: --caps-retain=CAP_SYS_ADMIN
+type appCapsRetain apps.Apps
+
+func (au *appCapsRetain) Set(s string) error {
+	app := (*apps.Apps)(au).Last()
+	if app == nil {
+		return fmt.Errorf("--caps-retain must follow an image")
+	}
+	capsRetain, err := types.NewLinuxCapabilitiesRetainSet(strings.Split(s, ",")...)
+	if err != nil {
+		return err
+	}
+	app.CapsRetain = capsRetain
+	return nil
+}
+
+func (au *appCapsRetain) String() string {
+	app := (*apps.Apps)(au).Last()
+	if app == nil {
+		return ""
+	}
+	var vs []string
+	for _, v := range app.CapsRetain.Set() {
+		vs = append(vs, string(v))
+	}
+	return strings.Join(vs, ",")
+}
+
+func (au *appCapsRetain) Type() string {
+	return "appCapsRetain"
+}
+
+// appCapsRemove is for --caps-retain flags in the form of: --caps-retain=CAP_SYS_ADMIN
```

Comment:
`appCapsRemove is for --caps-remove flags in the form of: --caps-remove=CAP_SYS_ADMIN`